lib/jekyll/commands/serve.rb

```diff
@@ -33,6 +33,8 @@ def init_with_program(prog)
               opts["serving"] = true
               opts["watch"  ] = true unless opts.key?("watch")
               config = opts["config"]
+              default_url = "http://localhost:4000"
```

Comment:
@parkr Thanks! Should i also take `baseurl` and `ssl` options into consideration?